### PR TITLE
Fix typo in README

### PR DESCRIPTION
**Pull request by udoyen**
### What type of PR is this?

documentation

### What this PR does / Why we need it?

Improves the documentation

### Which Jira/Github issue(s) does this PR fix?

_Resolves #_

### Special notes for your reviewer

Did a minor update

### Pre-checks (if applicable)

- [ ] Ran unit tests locally
- [ ] Validated the changes in a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

File: README.md~
Comment (mjlshen) [inline README.md~:1]: Thanks for the PR! Could you remind removing a couple extra files from this PR? This file and `.README.md.un~`?

[the remark this answers is not in the document]
Comment (udoyen) [reply README.md~:1]: Sure will do that

Comment (udoyen) [reply README.md~:1]: sure

**Review by supreeth7 — COMMENTED** [review]
/ok-to-test

**Pull request merged**